vector/src/main/java/im/vector/app/core/ui/views/KeysBackupBanner.kt

```diff
@@ -191,61 +163,12 @@ class KeysBackupBanner @JvmOverloads constructor(
         views.viewKeysBackupBannerLoading.isVisible = false
     }
 
-    /**
-     * The state representing the view.
-     * It can take one state at a time.
-     */
-    sealed class State {
-        // Not yet rendered
-        object Initial : State()
-
-        // View will be Gone
-        object Hidden : State()
-
-        // Keys backup is not setup, numberOfKeys is the number of locally stored keys
-        data class Setup(val numberOfKeys: Int) : State()
-
-        // Keys backup can be recovered, with version from the server
-        data class Recover(val version: String) : State()
-
-        // Keys backup can be updated
-        data class Update(val version: String) : State()
```

Comment:
This has been moved to the class `BannerState` in the ViewModel.

@BillCarsonFr this is worth noting that `Recover` and `Update` are actually never used.